libs/cart/state/src/facades/cart/cart.facade.ts

```diff
@@ -317,18 +339,46 @@ export class DaffCartFacade<
 		return this.store.pipe(select(this._selectCartItemCompositeOptions, { id: itemId }));
 	};
 
-	getCartItemDiscountedTotal(itemId: U['item_id']): Observable<number> {
-		return this.store.pipe(select(this._selectCartItemDiscountedRowTotal, { id: itemId }));
-	}
-
 	isCartItemOutOfStock(itemId: U['item_id']): Observable<boolean> {
-		return this.store.pipe(select(this._selectIsCartItemOutOfStock, { id: itemId }));
+    return this.store.pipe(select(this._selectIsCartItemOutOfStock, { id: itemId }));
 	}
 
 	getCartItemState(itemId: U['item_id']): Observable<DaffCartItemStateEnum> {
-		return this.store.pipe(select(this._selectCartItemState, { id: itemId }));
+    return this.store.pipe(select(this._selectCartItemState, { id: itemId }));
 	}
 
+	getCartItemTotalDiscount(itemId: U['item_id']): Observable<number> {
+    return this.store.pipe(select(this._selectCartItemTotalDiscount, { id: itemId }))
+  }
+
+	getCartItemDiscount(itemId: U['item_id']): Observable<number> {
+    return this.store.pipe(select(this._selectCartItemDiscount, { id: itemId }))
+  }
+
+	getCartItemHasDiscount(itemId: U['item_id']): Observable<boolean> {
+    return this.store.pipe(select(this._selectCartItemHasDiscount, { id: itemId }))
+  }
+
+	getCartItemPrice(itemId: U['item_id']): Observable<number> {
+    return this.store.pipe(select(this._selectCartItemPrice, { id: itemId }))
+  }
+
+	getCartItemDiscountedPrice(itemId: U['item_id']): Observable<number> {
+    return this.store.pipe(select(this._selectCartItemDiscountedPrice, { id: itemId }))
+  }
+
+	getCartItemQuantity(itemId: U['item_id']): Observable<number> {
+    return this.store.pipe(select(this._selectCartItemQuantity, { id: itemId }))
+  }
+
+	getCartItemRowTotal(itemId: U['item_id']): Observable<number> {
+    return this.store.pipe(select(this._selectCartItemRowTotal, { id: itemId }))
+  }
+
+  getCartItemDiscountedTotal(itemId: U['item_id']): Observable<number> {
```

Comment:
I think the facade name should match the selector name: `getCartItemDiscountedRowTotal`

Reply:
changed